--- bibtex_js.orig.js	2010-06-19 14:36:22.000000000 -0400
+++ bibtex_js.js	2014-05-27 21:34:16.000000000 -0400
@@ -153,7 +153,7 @@
         throw "Runaway key";
       }
     
-      if (this.input[this.pos].match("[a-zA-Z0-9_:\\./-]")) {
+      if (this.input[this.pos].match("[a-zA-Z0-9_:\\./+-]")) {
         this.pos++
       } else {
         return this.input.substring(start, this.pos).toUpperCase();
@@ -388,4 +388,4 @@
 
     bibtex_js_draw();
   });
-}
\ No newline at end of file
+}
